repondeur/zam_repondeur/fetch/an/amendements.py

```diff
@@ -219,9 +219,17 @@ def get_groupe(amendement: OrderedDict) -> str:
     if int(auteur["estGouvernement"]) or "@xsi:nil" in auteur["groupeTribunId"]:
         return ""
     groupes = get_data("organes")
-    groupe = groupes[f"PO{auteur['groupeTribunId']}"]
-    libelle: str = groupe["libelle"]
-    return libelle
+    try:
+        groupe_tribun_id = f"PO{auteur['groupeTribunId']}"
+    except KeyError:
+        logger.error(
+            "Unknown groupe %r for amendement %s",
+            groupe_tribun_id,
+            amendement["numero"],
+        )
+        return ""
+    groupe: Dict[str, str] = groupes[groupe_tribun_id]
```

Comment:
Are we sure that line and the following one will not raise a `KeyError` at some point too?

Reply:
We've never met this case... yet. 😉